classes/class-wc-rest-connect-self-help-controller.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! defined( 'WOOCOMMERCE_CONNECT_MAX_SELF_HELP_DEPTH' ) ) {
+	define( 'WOOCOMMERCE_CONNECT_MAX_SELF_HELP_DEPTH', 32 );
```

Comment:
Sure - I'll create a single constant for all these json_decode limiters and use it throughout WCC